apps/webapp/vite.config.ts

```diff
@@ -4,4 +4,8 @@ import react from '@vitejs/plugin-react-swc';
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
+  test: {
+    globals: true,
+    environment: 'jsdom',
+  },
```

Comment:
What specifically requires this? It should have a code comment that explains why it's necessary if so.